test/util/e2e.go

```diff
@@ -446,6 +446,14 @@ func WaitForPodRunning(ctx context.Context, k8sClient client.Client, pod *corev1
 	}, LongTimeout, Interval).Should(gomega.Succeed())
 }
 
+func IsOperatorRunning() bool {
+	value, found := os.LookupEnv("OPERATOR_RUNNING")
```

Comment:
Is this valid as is right now?

ie is that environment already set?

Reply:
would be set here https://github.com/openshift/kueue-operator/pull/538/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R235

Reply:
Is there a way to have this run on in this CI also?